xlcalculator/xlfunctions/func_xltypes.py

```diff
@@ -279,7 +279,7 @@ def __le__(self, other):
 
     def __eq__(self, other):
         if self.value in (None, '') \
-                and (other in (None, '') or isinstance(other, None)):
+                and (other in (None, '') or other is None):
```

Comment:
This simply duplicates in the "contains" expression before thus making it superfluous.